src/AbstractBuilder.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace Staffim\SpreadsheetBuilder;
+
+use PhpOffice\PhpSpreadsheet\Spreadsheet;
+
+abstract class AbstractBuilder
```

Comment:
We definitely should have one parent for worksheet builders `AbstractWorksheetBuilder` and two child classes: `AbstractTableWorksheetBuilder` and `TemplateWorksheetBuilder`  and they should return same result. For example, we should have ability to use them for building one file:

```php
use Staffim\SpreadsheetBuilder\Builder;

/// ....

$builder = new Builder([
    new \Acme\Xlsx\UsersListWorksheetBuilder(),
    new \Staffim\SpreadsheetBuilder\TemplateWorksheetBuilder('/path/to/file.xlsx', 'Second worksheet name', 'A1:Z10'),
]);
```

right now its impossible because of this: https://github.com/staffim/spreadsheet-builder/blob/master/src/Builder.php#L26